flang/lib/Optimizer/Transforms/AbstractResult.cpp

```diff
@@ -264,8 +265,7 @@ class AbstractResultOpt : public fir::AbstractResultOptBase<AbstractResultOpt> {
         return true;
       auto resultType = dispatch->getResult(0).getType();
       if (resultType.isa<fir::SequenceType, fir::BoxType, fir::RecordType>()) {
-        mlir::emitError(dispatch.getLoc(),
-                        "TODO: dispatchOp with abstract results");
+        TODO(dispatch.getLoc(), "TODO: dispatchOp with abstract results");
```

Comment:
You may want to remove the `TODO:` in the message.

Reply:
Good catch.  Will do.